### ISSUE TITLE
Espurna + Blynk server

### ISSUE THREAD

**Issue by artemicruz** [xoseperez/espurna#559]
It would be wonderful to add blynk server connection.
add blynk libraries are open source
the user only needs: server, port and tocken (online or private) 

http://docs.blynk.cc/#hardware-set-ups-esp8266-standalone

**Comment by thaeger71** [comment]
Is someone working on this this ? Otherwise I would start to work on it .... but just for case there is already something, I would offer my help.

**Comment by xoseperez** [comment]
There is no active development on this, at least not in the main repo.

**Comment by thaeger71** [comment]
Hey guys,

I forked the master branch and implented Blynk support. How is the correct way to get it into the regular branch ? Do I have to make a pull request ?

**Comment by xoseperez** [comment]
The code looks mostly OK. I have not tested it yet. But I would not enable Blynk by default unless someone points out a very good reason to do it.

**Comment by bluexp1** [comment]
Is there any timeline to release this feature?

**Comment by thaeger71** [comment]
It's a good question, I mean Max made also some effort to optimze things, now everthing seems to work, but so far I know there is no feedback from Xose on this so far ....

**Comment by mcspr** [comment]
Yep. #1230 PR is still pending (+ maybe needs another thought about wiring more things... and how that wiring is happening)
I have not used it very much though.

**Comment by thaeger71** [comment]
I marked everything pending as solved regarding #1230 is there something I have to in addition from my side ?

**Comment by xoseperez** [comment]
I'm going to release 1.13.4 first and then merge the code into `dev`.
